common/djangoapps/util/milestones_helpers.py

```diff
@@ -9,13 +9,19 @@
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
 
+from milestones import api as milestones_api
```

Comment:
It would be great if we can move this import here, but we should make sure we don't run into any circular import issues. Please be sure to exercise features that use this module.